aeron-archive/src/main/java/io/aeron/archive/ReplicationSession.java

```diff
@@ -481,7 +496,7 @@ else if (epochClock.time() >= (timeOfLastActionMs + actionTimeoutMs))
                     throw new ArchiveException("cannot live merge without active source recording");
                 }
 
-                state(State.EXTEND);
+                state(State.EXTEND, "srcRecordingPosition");
```

Comment:
We should either include value of the `srcRecordingPosition` into the message or maybe use empty string, i.e. as the state transition `SRC_RECORDING_POSITION -> EXTEND` is self explanatory.